python/packages/azurefunctions/agent_framework_azurefunctions/_durable_agent_state.py

```diff
@@ -0,0 +1,1233 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+"""Durable agent state management conforming to the durable-agent-entity-state.json schema.
```

Comment:
NIT: Couldn't find any file named `durable-agent-entity-state.json`, is that supposed to be added?